client/reader/full-post/index.jsx

```diff
@@ -53,6 +53,7 @@ var CommentButton = require( 'components/comment-button' ),
 
 import PostExcerpt from 'components/post-excerpt';
 import { getPostTotalCommentsCount } from 'state/comments/selectors';
+import GalleryView from 'components/tinymce/plugins/wpcom-view/gallery-view';
```

Comment:
The `GalleryView` guts are already extracted to a common component: [`<GalleryShortcode />`](https://github.com/Automattic/wp-calypso/tree/master/client/components/gallery-shortcode). Should probably be used here instead.